calicoctl/commands/config.go

```diff
@@ -317,3 +337,38 @@ func (a asnum) get(node string) error {
 	fmt.Println(asn.String())
 	return nil
 }
+
+// felixconfig implements the configType interface.
+type felixconfig struct {
+	name string
+	c client.ConfigInterface
+}
+
+func (f felixconfig) set(value, node string) error {
+	if node != "" {
+		return errors.New("--node should not be specified")
```

Comment:
Why can't you specify a node?  That can be really useful to turn on logging for one host only